haskell-style.md

```diff
@@ -593,14 +593,22 @@ module Driver
   ```yaml
   default-extensions:
     - BangPatterns
+    - DeriveAnyClass
+    - DeriveFoldable
+    - DeriveFunctor
     - DeriveGeneric
+    - DeriveLift
+    - DeriveTraversable
+    - DerivingStrategies
+    - GeneralizedNewtypeDeriving
     - LambdaCase
     - NoImplicitPrelude
     - NoMonomorphismRestriction
     - OverloadedStrings
     - RankNTypes
     - RecordWildCards
     - ScopedTypeVariables
+    - StandaloneDeriving
```

Comment:
I'm not sure about this one. I don't think it totally aligns with the stated motives of this PR, and (IMO) it promotes Orphans. Would you consider merging without it?

Reply:
I think we would catch orphans in code review, plus a submitter would have to manually suppress the warning. The other use case for this is deriving instances for GADTs which don't support stock deriving. 

The parser catches you doing it when it's turned off so I don't think it's adding any overhead

Reply:
Yeah, standalone deriving isn't about orphans, it is just about being able to enumerate extra constraints that GHC can't infer in normal deriving.